.github/workflows/build.yml

```diff
@@ -30,3 +30,9 @@ jobs:
 
       - name: Run build
         run: make dist
+
+      - name: Check HTML
+        uses: anishathalye/proof-html@v1
```

Comment:
I'm confused. Why is a different tool and not `html -validate` used here?

Reply:
@hanzei I've made the changes. Now, the workflow also uses `html-validate`.